src/avfoundation.cs

```diff
@@ -11350,6 +11844,10 @@ interface AVContentKeySession {
 		[Export ("renewExpiringResponseDataForContentKeyRequest:")]
 		void RenewExpiringResponseData (AVContentKeyRequest contentKeyRequest);
 
+		[NoWatch, NoTV, NoMac, iOS (11,0)]
+		[Export ("makeSecureTokenForExpirationDateOfPersistableContentKey:completionHandler:")]
+		void MakeSecureToken (NSData persistableContentKeyData, Action<NSData, NSError> handler);
```

Comment:
Forget the above but double check the generated code, it should return a `Task` (not `void`, so not an issue for bindings).